### DISCUSSION TITLE
BeeRef 0.2.0 (2021-09-06)

### DISCUSSION THREAD

**Discussion by rbreu** [rbreu/beeref#24] [Anncouncements]
This version enables text support. Note that bee files from version 0.2.0 won't open in BeeRef 0.1.x.

### Added

* You can now add plain text notes and paste text from the clipboard
* You can now open bee files from finder on MacOS (by David Andrs)
* Dragging bee files onto an empty scene will now open them
* Adding the first image(s) to a new scene centers the view

### Changed

* Make debug log file less verbose
* The program is now bundled as a single executable file (issue #4)

### Fixed

* Hovering over the scale handles of very narrow items now displays
  correct cursor orientation
* Fix a rare crash while displaying selection handles

<hr /><em>This discussion was created from the release <a href='https://github.com/rbreu/beeref/releases/tag/v0.2.0'>BeeRef 0.2.0 (2021-09-06)</a>.</em>

**Comment by in3des** [comment]
Hi!
cannot install it on win7x64 
![image](https://user-images.githubusercontent.com/62700347/132400986-cf2be941-1129-4961-87a1-6b931788aa91.png)

**Reply by rbreu** [reply]
Windows 7 is deprecated.

**Comment by myselfhimself** [comment]
Succesfully installed on Linux (using the Binary release)

**Comment by yuk159** [comment]
Successfully installed on Linux and thank you very much it's really a great tool for an illustrator like me... :)

**Comment by audiovisualboy** [comment]
On macOS Monterey 12.5.1 here. Instead of getting the "MacOS cannot verify developer ... etc" I'm getting this which is odd to me:  
<img width="399" alt="Screen Shot 2022-09-06 at 3 00 46 AM" src="https://user-images.githubusercontent.com/20306002/188558249-bdebc63c-9380-4bb7-b42c-d8808e6c686a.png">
